packages/components/src/box-control/index.tsx

```diff
@@ -189,12 +186,24 @@ function BoxControl( {
 				</LinkedButtonWrapper>
 			) }
 
-			{ ! isLinked && splitOnAxis && (
-				<AxialInputControls { ...inputControlProps } />
-			) }
-			{ ! isLinked && ! splitOnAxis && (
-				<InputControls { ...inputControlProps } />
-			) }
+			{ ! isLinked &&
+				splitOnAxis &&
+				[ 'horizontal', 'vertical' ].map( ( axis ) => (
```

Comment:
The axial controls appear to be listed in the opposite order compared to trunk.

Trunk:
![Screenshot 2024-12-06 at 13 00 41](https://github.com/user-attachments/assets/3ed3e326-9257-42b3-b6b7-2d5fe1a3fe59)



This branch:

![Screenshot 2024-12-06 at 13 00 35](https://github.com/user-attachments/assets/c2d1ffd4-e757-4f59-b373-40fb6c736ba3)

Should we just rearrange them in the array?